ecl/hql/hqlparse.cpp

```diff
@@ -1997,6 +1997,14 @@ int HqlLex::processStringLiteral(attribute & returnToken, char *CUR_TOKEN_TEXT,
                 finger++;
                 continue;  // A \ at end of line in a multiline constant means remove the end-of-line
             }
+            else if (next == '\r')
```

Comment:
Feels like something bigger is wrong if a multiline string literal from a Windows source file has a different contents than from a non-windows one. Isn't a more correct fix to have translate \r\n to \n earlier?